crates/fraud_proofs/global_merkle_root/storage/src/update.rs

```diff
@@ -138,11 +142,38 @@ where
         // TODO(#2584): Insert state transition bytecode and consensus parameter updates.
         // TODO(#2585): Insert uplodade bytecodes.
         // TODO(#2586): Insert blobs.
-        // TODO(#2587): Insert raw code for created contracts.
+        if let Transaction::Create(tx) = tx {
+            self.process_create_transaction(tx)?;
+        }
 
         Ok(())
     }
 
+    // CreateBody is not publicly exported, so we need to use a generic type
+    // and ensure that all trait bounds are satisfied
+    fn process_create_transaction(&mut self, tx: &Create) -> anyhow::Result<()> {
+        let bytecode_witness_index = tx.bytecode_witness_index();
+        let witnesses = tx.witnesses();
+        let bytecode = witnesses[usize::from(*bytecode_witness_index)].as_vec();
+        // The Fuel specs mandate that each create transaction has exactly one output of type `Output::ContractCreated`.
+        // See https://docs.fuel.network/docs/specs/tx-format/transaction/#transactioncreate
+        let Some(Output::ContractCreated { contract_id, .. }) = tx
+            .outputs()
+            .iter()
+            .filter(|output| matches!(output, Output::ContractCreated { .. }))
+            .take(1)
+            .collect::<Vec<&Output>>()
+            .first()
```

Comment:
Nit: I think this can be simplified
```suggestion
            .next()
```

Reply:
always nice to learn shortcuts :)